ash/src/extensions/khr/wayland_surface.rs

```diff
@@ -43,7 +43,7 @@ impl WaylandSurface {
         &self,
         physical_device: vk::PhysicalDevice,
         queue_family_index: u32,
-        wl_display: &mut vk::wl_display,
+        wl_display: *mut vk::wl_display,
```

Comment:
We should write these down as breaking changes in the `CHANGELOG` (maybe via a separate PR so that it shows up properly in the commit history), though if someone passes exactly a `&mut vk::wl_display` it'll automatically coerce to the raw pointer.